gym_ergojr/envs/ergo_reacher_augmented.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import torch
+from gym_ergojr.envs.ergo_reacher_env import ErgoReacherEnv
+from gym_ergojr.models.networks import LstmNetRealv1
+
+
+class ErgoReacherAugmentedEnv(ErgoReacherEnv):
+    def __init__(self, headless=False,
+               simple=False,
+               backlash=False,
+               max_force=1,
+               max_vel=18,
+               goal_halfsphere=False,
+               multi_goal=False,
+               goals=3,
+               gripper=False):
+        self.hidden_layers = 128
+        self.lstm_layers = 3
+        self.model = LstmNetRealv1(
+                n_input_state_sim=12,
+                n_input_state_real=12,
+                n_input_actions=6,
+                nodes=self.hidden_layers,
+                layers=self.lstm_layers)
+        self.model_path = '/home/sharath/neural-augmented-simulator/nas/data/model-exp1-h128-l3-v01-e5.pth'
```

Comment:
Could you put the model into this repo and then use a relative path, please. The folder `gym_ergojr/trained_lstms` is exactly for that purpose. You can use path like this:
```suggestion
        self.model_path = '"../trained_lstms/ergoreacher-exp1-h128-l3-v01-e5.pth'
```